src/main/java/frc/robot/subsystems/Feedback.java

```diff
@@ -14,18 +24,136 @@ public class Feedback extends SubsystemBase {
+    config.brightnessScalar = 0.5; // dim the LEDs to half brightness
+    bracelet.configAllSettings(config);
+    this.off();
+    this.setDefaultCommand(this.setAllianceColor());
+  }
+
+  public Command setAllianceColor() {
+    return this.run(
+        () -> {
+          var a = DriverStation.getAlliance();
+          if (a.isPresent() & a.get().equals(Alliance.Red)) {
+            setMulti(Color.kRed);
+          } else {
+            setMulti(Color.kBlue);
+          }
+        });
+  }
+
+  public Command shooterWheelsAtSpeed() {
+    return this.startEnd(
+      () -> {
+        StrobeAnimation sa = new StrobeAnimation(255, 0, 0, 255, 0.5, 60, 8);
+        bracelet.clearAnimation(1); 
+        bracelet.clearAnimation(2); 
+        bracelet.animate(sa, 1); 
+      }, this::off); 
+  }
+
+  public Command noteInCartridge() {
+    return this.startEnd(
+      () -> {
+        bracelet.clearAnimation(1);
+        bracelet.clearAnimation(2);
+        bracelet.setLEDs(255, 0, 0); 
+      }, this::off); 
+  }
+
+  public Command rainbows() {
+    return this.startEnd(this::animate, this::off);
+  }
+
+  public Command intakeCurrentSpike() {
+    return this.startEnd(
+      () -> {
+        bracelet.clearAnimation(1); 
+        bracelet.clearAnimation(2); 
+        bracelet.setLEDs(255, 255, 0); 
+      }, this::off);
+  }
+
+  public Command turnOffLEDs() {
+    return this.startEnd(
+      () -> {
+        bracelet.clearAnimation(1); 
+        bracelet.clearAnimation(2); 
+        bracelet.setLEDs(0, 0, 0);
+      }, this::off); 
+  }
+
+  public void disabledColorPattern() {
+    ColorFlowAnimation cfa = new ColorFlowAnimation(0, 0, 255, 255, 0.5, 68, Direction.Forward, 5);
+    bracelet.clearAnimation(1);
+    bracelet.clearAnimation(2);
+    bracelet.animate(cfa, 1);
+    ColorFlowAnimation cfa2 =
+        new ColorFlowAnimation(255, 0, 0, 255, 0.5, 68, Direction.Backward, 0);
+    bracelet.animate(cfa2, 2);
+  }
+
+  public void animate() {
+    RainbowAnimation rb = new RainbowAnimation();
+    bracelet.clearAnimation(1);
+    bracelet.clearAnimation(2);
+    rb.setBrightness(255);
+    rb.setLedOffset(0);
+    rb.setNumLed(300);
+    rb.setSpeed(255);
+    bracelet.animate(rb, 1);
   }
 
   public void red() {
-    bracelet.setLEDs(255, 0, 0);
+    setMulti(Color.kRed);
   }
 
-  public void multi(Color color) {
+  public void setMulti(Color color) {
+    bracelet.clearAnimation(1);
+    bracelet.clearAnimation(2);
+    bracelet.setLEDs(
+        ((int) (color.red * 255)),
+        ((int) (color.green * 255)),
+        ((int) (color.blue * 255)),
+        127,
+        0,
+        8);
     bracelet.setLEDs(
-        ((int) (color.red * 255)), ((int) (color.green * 255)), ((int) (color.blue * 255)));
+        ((int) (color.green * 255)),
+        ((int) (color.red * 255)),
+        ((int) (color.blue * 255)),
+        127,
+        8,
+        250);
+  }
+
+  public Command multi(Color color) {
```

Comment:
Can we call this something like `setColor`? `multi` makes it sound like the color output will be multi colored

Reply:
yes